### DISCUSSION TITLE
Организация Zapret как шлюз домашней сети

### DISCUSSION THREAD

**Discussion by djmagin** [bol-van/zapret#927] [Q&A]
Здравствуйте,  Идея сделать шлюз (linux 2 физических интерфейса LAN, WAN) для обхода ограничений ютуба и дискорда, есть ли подробная инструкция для установки/настройки? Если есть ткните меня носом, если нет то опишите пожалуйста процесс установки/настройки

**Comment by bol-van** [comment]
Инструкций как сделать шлюз на Linux в инете полно.
После этого zapret ставится обычным образом как на linux. Выбираются интерфейсы lan и wan в инсталляторе

**Reply by KweezyCode** [reply]
есть ли инструкция для openwrt как шлюз? я использую обычный установщик запрета, но почему-то не работает, openwrt-шлюз на айпишнике 192.168.1.100, curl с него до ntc.party работает, а клиенты таймаут ловят. Использую raspberry pi с одним лан портом

**Reply by bol-van** [reply]
Много раз вопрос поднимался. Используйте masquerade. иначе обратный трафик идет напрямую

**Comment by Igorka82** [comment]
Залежалась на работе Терминальная станция Kraftway Credo VV22. Заказал на али вифи Atheros QCA9880 5Г. Поставил openwrt. На него запрет. Все девайсы в доме на вифи. Все блокировки сняты.

**Comment by Geskawary234** [comment]
Ну как получилось что-то? Я щас хочу что-то такое провернуть с BigTreeTech PI v1.2

**Reply by HentayHentay** [reply]
Организовать шлюз можно на чем угодно
Дело в удобстве эксплуатации и настроек

**Reply by djmagin** [reply]
Да, сделал на orangepi zero 3, все работает кроме дс, но он мне и не нужен. [https://github.com/bol-van/zapret/discussions/2088](url) тут я рассказал как

**Reply by Geskawary234** [reply]
> Да, сделал на orangepi zero 3, все работает кроме дс, но он мне и не нужен. [https://github.com/bol-van/zapret/discussions/2088](url) тут я рассказал как

спасибо, гляну

**Reply by Geskawary234** [reply]
> Организовать шлюз можно на чем угодно Дело в удобстве эксплуатации и настроек

ну эт понятно, просто я не особо шарю за сети и все такое. Если кто-то делал то уже легче по примеру

**Reply by HentayHentay** [reply]
> не особо шарю за сети

Изучайте. Все есть у Гугла и Яндекса 

> кто-то делал то уже легче

Проблема легче в том, что иногда проще найти устаревшие настройки. Конечно можно и более современное.
Но начинать нужно как минимум  с выбора управления (ОС) и задачи
Тема к сожалению достаточно обширна и каждый лепит свое, со своими удобствами.

Лепить горбатого можно и без железо. Благо виртуализация у нас в чайнике ). Но там свои проблемы и трудности
